specifications/transactions/README.md

```diff
@@ -0,0 +1,61 @@
+# Diem Transactions and Payments
+
+## Overview
+
+The Diem Payment network supports transactions between VASPs (Virtual Asset Service Providers) and between VASPs and DDs (designated dealers). This document covers the end-to-end flow for these transactions derived from the DIPs.
+
+Diem supports two means for transacting on-chain:
+* Subaddresses that can be exchanged directly between two parties off-chain
+* ReferenceIds that can be generated via an off-chain API between two parties
+
+In the case of subaddresses, a participant can directly provide a subaddress to one or more other participants, who can then submit transactions directly to that party on-chain.
+
+However, re-use of subaddresses creates on-chain linkage and may constitute publicly identifiable material. To address this, participants can also leverage an off-chain API to produce a one-time use reference id for the on-chain payment.
+
+## Introduction
+
+### On-Chain
+
+A Diem transaction consists of a sender, a script, and a set of arguments. Currency transfer transactions contain a set of arguments including a recipient, an amount, a currency, and metadata. On success, the transaction would debit the sender's account by the specified amount in addition to gas (network usage) costs and credit the recipient's account. On failure, the transaction may entirely be discarded or committed to the blockchain in which case the sender would only be debited gas costs. Diem leverages the metadata to provide context behind a transaction. However, the metadata itself is not validated by the Diem blockchain or Move standard library.
```

Comment:
The sentence starting "On failure" is confusing because if theres a failure why would it be committed?

Also the last sentence starting "Diem leverages" What is "Diem" in this case? The blockchain, the payment network, its ambiguous and confusing cause Diem can have a couple meanings.